daily-challenges/2024-10/26/khanh5923_26.spec.ts

```
@@ -0,0 +1,47 @@
+import { test, expect, Page } from "@playwright/test";
+interface Product {
+  name: string | null;
+  qnt: string | null;
+}
+async function buyProduct(productsList: Product[], page: Page): Promise<void> {
+  let productSecondtList:Product[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'productSecondtList' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)